Gems/Multiplayer/Registry/prefab.tools.setreg

```diff
@@ -18,8 +18,14 @@
                         "GameObjectCreation":
                         [
                             { "$type": "AzToolsFramework::Prefab::PrefabConversionUtils::EditorInfoRemover" },
-                            { "$type": "Multiplayer::NetworkPrefabProcessor" },
-                            { "$type": "AzToolsFramework::Prefab::PrefabConversionUtils::PrefabCatchmentProcessor" }
+                            { 
+                                "$type": "Multiplayer::NetworkPrefabProcessor",
+                                "SerializationFormat": "Binary" // Options are "Binary" (default) or "Text". Prefer "Binary" for performance.  
+                            },
+                            { 
+                                "$type": "AzToolsFramework::Prefab::PrefabConversionUtils::PrefabCatchmentProcessor",
+                                "SerializationFormat": "Binary" // Options are "Binary" (default) or "Text". Prefer "Binary" for performance. 
```

Comment:
It's more that "Binary" is the default option and it's already declared in the base version so users can find it. Then again since it mirrors the new option you added and it's probably common to adjust both at the same time for debugging, you got a point, so lets keep it as-is then.